.github/workflows/validatePR.yml

```diff
@@ -0,0 +1,16 @@
+name: validate pr
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  pr-validation:
+    uses: salesforcecli/github-workflows/.github/workflows/validatePR.yml@main
+  format-check: 
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install modules
+      run: yarn
+    - name: Check Format
+      run: yarn format
```

Comment:
I think you'll need to add a yarn script to check the format vs just doing the usual --write. I ended up going with debug-check so it would tell us what's wrong in schemas: https://github.com/forcedotcom/schemas/blob/0adfecf208f85fafee44a19c165e9332fb2b11ab/package.json#L18